### PR TITLE
Feat notes page

### LINKED ISSUE #8: Notes Page

### PR DESCRIPTION
**Pull request by coldpigli**
Added note page.
Added Quill Editor
Setup up contexts and reducer for notes

### CODE REVIEW

File: src/components/WriteNote/WriteNote.module.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+.bottomToolbar{
+    display: flex;
+    width: 100%;
+    padding: 1rem;
+    align-items: center;
+    justify-content: center;
+    gap: 3rem;
+    color: rgba(8, 8, 8, 0.61);
```
Comment (rahulcs754) [inline src/components/WriteNote/WriteNote.module.css:47]: use variable for color

**Review by rahulcs754 — APPROVED** [review]
The code looks good and clean.
Design of page looks good.
just small suggestion

**Review by raman1999 — APPROVED** [review]
code looks clean to me  ,good component breakdown, you can merge

File: src/components/WriteNote/WriteNote.module.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+.notepad{
+    display: flex;
+    flex-direction: column;
+    width: 60%;
+    padding: 1rem;
+    margin: 0 auto;
+    position: relative;
+    background-color: #FFED7D;
```
Comment (srejitk) [inline src/components/WriteNote/WriteNote.module.css:36]: Use Variables for Color instead of using hexcodes

**Review by srejitk — APPROVED** [review]
No changes suggested. Intuitive classnames and good work styling the Quill component

**Pull request merged**